docs/third-party-developers/extensibility/data-store/cart.md

```diff
@@ -0,0 +1,378 @@
+- `total_shipping_tax`- string containing the total shipping tax
+- `total_tax`- string containing the total tax
+- `total_price`- string containing the total price
+- `tax_lines`- array containing the tax lines
+
+#### _Example_
+
+```js
+const store = select( 'wc/store/cart' );
+const cartTotals = store.getCartTotals();
+```
+
+### getCartMeta
+
+Returns the cart meta from state.
+
+#### _Returns_
+
+`object`- The current cart meta. This will be an object with the following keys:
+
+- `updatingCustomerData`- boolean indicating if the customer data is being updated
+- `updatingSelectedRate`- boolean indicating if the selected rate is being updated
+- `isCartDataStale`- boolean indicating if the cart data is stale
+- `applyingCoupon`- string containing the coupon code being applied
+- `removingCoupon`- string containing the coupon code being removed
+
+#### _Example_
+
+```js
+const store = select( 'wc/store/cart' );
+const cartMeta = store.getCartMeta();
+```
+
+### getCartErrors
+
+Returns the cart errors from state.
+
+#### _Returns_
+
+`array`- array containing the cart errors.
+
+#### _Example_
+
+```js
+const store = select( 'wc/store/cart' );
+const cartErrors = store.getCartErrors();
+```
+
+### isApplyingCoupon
+
+Queries whether a coupon is being applied.
+
+#### _Returns_
+
+`boolean`- True if a coupon is being applied.
+
+#### _Example_
+
+```js
+const store = select( 'wc/store/cart' );
+const isApplyingCoupon = store.isApplyingCoupon();
+```
+
+### isCartDataStale
+
+Queries whether the cart data is stale.
+
+#### _Returns_
+
+`boolean`- True if the cart data is stale.
+
+#### _Example_
+
+```js
+const store = select( 'wc/store/cart' );
+const isCartDataStale = store.isCartDataStale();
+```
+
+### getCouponBeingApplied
+
+Returns the coupon code being applied.
+
+#### _Returns_
+
+`string`- The coupon code being applied.
+
+#### _Example_
+
+```js
+const store = select( 'wc/store/cart' );
+const couponBeingApplied = store.getCouponBeingApplied();
+```
+
+### isRemovingCoupon
+
+Queries whether a coupon is being removed.
+
+#### _Returns_
+
+`boolean`- True if a coupon is being removed.
+
+#### _Example_
+
+```js
+const store = select( 'wc/store/cart' );
+const isRemovingCoupon = store.isRemovingCoupon();
+```
+
+### getCouponBeingRemoved
+
+Returns the coupon code being removed.
+
+#### _Returns_
+
+`string`- The coupon code being removed.
+
+#### _Example_
+
+```js
+const store = select( 'wc/store/cart' );
+const couponBeingRemoved = store.getCouponBeingRemoved();
+```
+
+### getCartItem
+
+Returns a cart item from the state.
+
+#### _Parameters_
+
+- cartItemKey `string` - The cart item key.
+
+#### _Returns_
+
+`object`- The cart item. This will be an object with the following keys:
+
+- `key`- string containing the cart item key
+- `id`- number containing the cart item id
+- `catalog_visibility`- string containing the catalog visibility
+- `quantity_limits`- object containing the quantity limits
```

Comment:
Do you think it would be useful to describe the shape of these objects here? `prices` and `totals` too?